lib/ohai/plugins/linux/grub2.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+#
+# Author:: Davide Cavalca <dcavalca@fb.com>
+# Copyright:: Copyright (c) 2020 Facebook
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+Ohai.plugin(:Grub2) do
+  provides "grub2/environment"
+  optional true
+
+  collect_data(:linux) do
```

Comment:
I imagine this works fine on *bsd as well...

Reply:
uh, I guess? I think FreeBSD has grub in ports, but I don't recall ever seeing it on other flavors. I can certainly enable it, though I won't have an easy way to test it at the moment.

Reply:
Ok, it's in ports for {Dragonfly,Free,Net}BSD but not for OpenBSD.